packages/parser/src/lib/detectNoSlashPaths.ts

```diff
@@ -0,0 +1,34 @@
+import type { OpenAPIV2, OpenAPIV3, OpenAPIV3_1 } from 'openapi-types';
+
+/**
+ * Detects paths in the API definition that are missing leading slashes.
+ * This is a common issue that causes "ADDITIONAL PROPERTY" validation errors.
+ *
+ * @param api - The API definition to check
+ * @returns Array of paths that are missing leading slashes
+ */
+export function detectNoSlashPaths(api: OpenAPIV2.Document | OpenAPIV3_1.Document | OpenAPIV3.Document): string[] {
+  const noSlashPaths: string[] = [];
+
+  if (!api.paths) {
+    return noSlashPaths;
+  }
+
+  // Check each path in the paths object
+  Object.keys(api.paths).forEach(path => {
```

Comment:
As an additional level of protection you might want to only run `Object.keys()` if `api.paths` is an object. If they've given us anything else for this, like a boolean or an array, whatever, we should avoid crashing here and let the AJV parsing handle and return an appropriate error back to them.